data/groupedbaryw.json

```diff
@@ -0,0 +1,15 @@
+[{"id":0,"type":132,"xval":"Aug 20","yval":74.928,"name":"Total Average","breakdown":"1634","breakdowntype":"YW","parentrefid":163,"parentreftype":135,"zval":1300},
```

Comment:
I asked Jose about this in Jira. He clarified that this is a type and that the xvals shouldnt be a mix of dates and strings . I will update this Json on my next push